src/statement/delete.js

```diff
@@ -63,17 +62,19 @@ class Delete extends Statement {
       throw new Error("Invalid `DELETE` statement, missing `FROM` clause.");
     }
 
-    return 'DELETE' +
+    const dialect = this.dialect()
+    return (DELETE +
```

Comment:
we could probably eek out a little less memory usage by using an array and join'ing at the end. I did a lot of work on util.inspect a little while back and Array#join() was on par performance wise, but had lower memory usage than a template string (or a lot of little string concatenations). May not be an issue in this code path, just something to think about.

Reply:
maybe not this one, but select + update would be. I can do that